hazelcast/src/hazelcast/client/map/impl/DataAwareEntryEvent.cpp

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "hazelcast/client/map/impl/DataAwareEntryEvent.h"
+
+namespace hazelcast {
```

Comment:
Is this file leftover ?

Reply:
Yes, it was left over and I just deleted this left-over file.